pkg/providers/launchtemplate/suite_test.go

```diff
@@ -1117,7 +1117,7 @@ var _ = Describe("LaunchTemplates", func() {
 				ExpectApplied(ctx, env.Client, nodePool)
 				pod := coretest.UnschedulablePod()
 				ExpectProvisioned(ctx, env.Client, cluster, cloudProvider, prov, pod)
-				// This will not be scheduled since we were pointed to a non-existent awsnodetemplate resource.
+				// This will not be scheduled since we were pointed to a non-existent resource.
```

Comment:
nit: I think it's worth to have the ec2nodeclass in this comment, but up to you.